package.json

```diff
@@ -7,11 +7,11 @@
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
     "ad-bs-converter": "^0.5.0",
-    "node-sass": "4.14.1",
```

Comment:
I don't really know why, but this package was throwing me an error when trying to do "npm install". I had to delete it and replace it with "sass". No breaking changes.

Reply:
Looks good to me. Thanks